Tasks/AzureResourceGroupDeploymentV2/task.json

```diff
@@ -432,7 +432,7 @@
         "AdditionFailed": "Addition of %s on vm %s failed. Error : %s",
         "DeletionFailed": "Deletion of %s on vm %s failed. Error : %s",
         "DeleteAgentManually": "After the task execution completes, if the deployment group agent corresponding to the virtual machine %s is present but not active in the deployment group %s, then please delete it manually from the deployment group.",
-        "DGAgentOperationOnAllVMsFailed": "Deployment group agent %s did not succeed on all VMs. %s",
+        "DGAgentOperationOnVMFailed": "Deployment group agent %s did not succeed on VM. Error: %s",
```

Comment:
Is this message really useful? Should we directly put the error string.

Reply:
Yes, can be done as VM extension error starts with "VM has reported a failure when processing extension 'TeamServicesAgent'"